src/core_atmosphere/physics/mpas_atmphys_driver_convection.F

```diff
@@ -986,13 +982,13 @@ subroutine convection_to_MPAS(configs,mesh,diag_physics,tend_physics,its,ite,exc
 
 !local variables:
  integer:: i,k,j
- integer:: iCell,ineighbor
+ integer:: iCell,ineighbor,ineighbor_far
 
 !local pointers:
  character(len=StrKIND),pointer:: convection_scheme
  integer,dimension(:),pointer:: k22_shallow,kbcon_shallow,ktop_shallow,ktop_deep
- real(kind=RKIND),dimension(:),pointer  :: nca,cubot,cutop,cuprec,raincv
- real(kind=RKIND),dimension(:),pointer  :: rainncv,vcpool,sigma_deep
+ real(kind=RKIND),dimension(:),pointer  :: nca,cubot,cutop,cuprec,rainncv,raincv,vcpool
+ real(kind=RKIND),dimension(:),pointer  :: sigma_deep,sigma_deep_far
```

Comment:
same as above, why not just add sigma_deep_far to the previous line 995?

Reply:
Updated. Thanks.